python/plugins/processing/algs/qgis/TinInterpolation.py

```diff
@@ -29,28 +29,31 @@
 
 from qgis.PyQt.QtGui import QIcon
 
-from qgis.core import QgsRectangle, QgsWkbTypes
+from qgis.core import QgsRectangle
 from qgis.analysis import (QgsInterpolator,
                            QgsTINInterpolator,
                            QgsGridFileWriter
-                           )
+                          )
 
 from processing.core.GeoAlgorithm import GeoAlgorithm
-from processing.core.parameters import ParameterVector
-from processing.core.parameters import ParameterSelection
-from processing.core.parameters import ParameterNumber
-from processing.core.parameters import ParameterExtent
-from processing.core.outputs import OutputRaster
-from processing.core.outputs import OutputVector
+from processing.core.GeoAlgorithmExecutionException import GeoAlgorithmExecutionException
+from processing.core.parameters import (Parameter,
+                                        ParameterNumber,
+                                        ParameterExtent,
+                                        ParameterSelection,
+                                        _splitParameterOptions
+                                       )
+from processing.core.outputs import (OutputRaster,
+                                     OutputVector
+                                    )
 from processing.tools import dataobjects
 
 pluginPath = os.path.split(os.path.split(os.path.dirname(__file__))[0])[0]
```

Comment:
Does this double-split look look strange to you as well?
(I realize that it wasn't changed in this pull request)